lib/stream-pyramid.js

```diff
@@ -129,19 +129,27 @@ Pyramid.prototype._read = function() {
         } else {
             return push(null) && false;
         }
-        function done(err, buffer) {
+        function done(err, buffer, headers) {
+            headers = headers || {};
             stream.pending--;
             if (err && !(/does not exist$/).test(err.message)) {
                 stream.emit('error', err);
             } else if (err || isEmpty(buffer)) {
                 tile = new Tile(z, x, y);
```

Comment:
Small style thing, this `tile = new Tile(z, x, y);` will read better for me within the first branch of the `if` since it's only used in that case.